index.go

```diff
@@ -295,39 +300,42 @@ func main() {
 		r.Get("/search/geo/:latitude/:longitude", dealers_ctlr.SearchLocationsByLatLng)
 	})
 
+	//Creating, updating, and deleting FAQs are done in GoAdmin directly
```

Comment:
Yes, we can most likely remove large chunks of this API with little to no effect